metagraph-plugin/README.md

```diff
@@ -36,25 +36,25 @@ pytest tests
 Examples
 ===========================
 
-Loading Katana Graph
+Loading An Example Graph
 ------------------
 
 ```
 import metagraph as mg
-import katana.local
-from katana.example_data import get_input
-katana.local.initialize()
-pg = katana.local.Graph(get_input('propertygraphs/rmat15_cleaned_symmetric'))
-katana_graph = mg.wrappers.Graph.KatanaGraph(pg)
+import pandas as pd
```

Comment:
Let's not use the contraction `pd`. It's not mentioned enough to actually shorten things much and contractions like this almost always may code less accessible to people who don't already know how to use these libraries.

Reply:
just use ```pandas```, right?

Reply:
Yep.